### PR TITLE
Updated UI

### PR DESCRIPTION
**Pull request by EStroiu**
Polished up admin panel, handler view, and forms both for queue and requests with bootstrap.

### CODE REVIEW

File: static/js/admin.js [hunk 1]
```diff
@@ -7,11 +7,15 @@ $(function() {
...
-      $("#info").append("<br>Public board can be found <a href='/public/board'>" + window.location.href.split("/")[2] + "/public/board</a>");
-
+      console.log(window.location.href.split("/")[2] + "/join/" + data.code);
+      let addEntryLink = window.location.href.split("/")[2] + "/join/" + data.code;
+      let handlerViewLink = window.location.href.split("/")[2] + "/join/handler/" + data.authcode;
+      let publicViewLink = window.location.href.split("/")[2] + "/public/board";
+      $("#info").html("<li class='nav-item'><a class='nav-link mx-3' href='http://" + addEntryLink + "' target='_blank'>" + "<b>Add Entry</b>" + "</a></li>");
+      $("#info").append("<li class='nav-item'><a class='nav-link mx-3' href='http://" + handlerViewLink + "' target='_blank'>" + "<b>Handler View</b>"+ "</a></li>");
+      $("#info").append("<li class='nav-item'><a class='nav-link mx-3' href='http://"+ publicViewLink +"' target='_blank'>" + "<b>Public Board</b>" +"</a></li>");
```
Comment (That2amCoder) [inline static/js/admin.js]: Why not add it as ``href='/<dir>'`` e.g "/join/handler/"? I don't see why you insert the domain manually

File: static/js/admin.js [hunk 2]
```diff
@@ -41,7 +45,12 @@ $(function() {
         // convert it to minutes
         var minutes = Math.floor(timeSince / 60000);
 
-        var entryHtml = "<div class='entry' id='entry-" + id + "'>" + name + "<br>" + question +"<br>" + minutes + " minutes ago. <br> Handled by: <b>" + handlerName + "</b></div>";
+        // var entryHtml = "<div class='entry' id='entry-" + id + "'>" + name + "<br>" + question +"<br>" + minutes + " minutes ago. <br> Handled by: <b>" + handlerName + "</b></div>";
```
Comment (That2amCoder) [inline static/js/admin.js]: Don't be afraid to delete code, don't leave them as comments, unless they are comments.

File: static/js/admin.js [hunk 3]
```diff
@@ -98,10 +147,10 @@ $(function() {
   }
 
   // Update the boxes every 30 seconds
-  setInterval(updateBoxes, 10000);
+  //setInterval(updateBoxes, 10000);
```
Comment (That2amCoder) [inline static/js/admin.js]:
```suggestion
```

File: static/js/handler-terminal.js [hunk 4]
```diff
@@ -15,15 +15,24 @@ $(function() {
...
-          entryHtml += "<button id='" + id + "' class='finished-button'>Finished</button>";
-          
+        // var entryHtml = "<p class='entry-name'>" + name + "</p>" +
+        //                 "<p>" + question + "</p>" +
+        //                 "<p>" + extra + "</p>" +
+        //                 "<p class='entry-time'>Submitted " + minutes + " minute(s) ago</p>";
+        // //Set the button IDs the same as the entry ID so that when the POST request happens, we know which entry to update
+        //   entryHtml += "<button id='" + id + "' class='ghost-button'>Ghost</button>";
+        //   entryHtml += "<button id='" + id + "' class='in-progress-button'>In Progress</button>";
+        //   entryHtml += "<button id='" + id + "' class='finished-button'>Finished</button>";
```
Comment (That2amCoder) [inline static/js/handler-terminal.js]:
```suggestion
```

File: static/templates/admin.html [hunk 5]
```diff
@@ -1,15 +1,51 @@
 <!DOCTYPE html>
```
Comment (That2amCoder) [inline static/templates/admin.html:1]: The admin page does not scale to 1080 screens (for some reason)

**Review by That2amCoder — CHANGES_REQUESTED** [review]
Overall a major upgrade in terms of UI! Thank you, however see attached comments.

File: static/js/admin.js (hunk 2 above)
Comment (EStroiu) [reply static/js/admin.js]: Will do.

File: static/js/admin.js (hunk 1 above)
Comment (EStroiu) [reply static/js/admin.js]: Oh, you mean relative URLs, yeah I think I see it now. I overcomplicated it a little bit, will change it.

**Pull request merged**